src/main/java/io/netty/buffer/api/memseg/BifurcatedDrop.java

```diff
@@ -50,7 +50,7 @@ void increment() {
     }
 
     @Override
-    public synchronized void drop(T obj) {
+    public synchronized void drop(MemSegBuf buf) {
```

Comment:
Not really related to this PR but I think we should try to reduce the `synchronised` scope. Who knows that the delegate does...

Reply:
Forgot to remove the synchronised after changing to using atomic ops.